README.rst

```diff
@@ -2,7 +2,7 @@ Object-oriented interface to `Service Abstraction Layer <https://docushare.lsstc
 
 `Documentation <https://ts-salobj.lsst.io>`_
 
-The package is compatible with setuptools, as well as the Vera Rubin LSST DM's ``eups`` package management system and ``scons`` build system.
+The package is compatible with setuptools, as well as the Vera C. Rubin LSST DM's ``eups`` package management system and ``scons`` build system.
```

Comment:
How about:
```markdown
The package is compatible with setuptools, as well as the [``eups`` package management system](https://github.com/RobertLuptonTheGood/eups) and ``scons`` build system.
```
?

Reply:
(except you are using rST and not markdown...)

Reply:
Sounds good, though I will make only "eups" the link.